pages/cart.js

```diff
@@ -0,0 +1,34 @@
+const { I } = inject();
+
+module.exports = {
+  emailInput: { css: '#email' },
+  passwordInput: { css: '#passwd' },
+  submitLogin: { css: '#SubmitLogin' },
+  agreement: { css: '#cgv' },
+  productPrice: { css: '#our_price_display' },
+  finalPrice: { css: '#total_price' },
+  hookPayment: {xpath: '//*[@id="HOOK_PAYMENT"]/div[1]/div/p/a'},
+
+
+  async proceedToCart() {
+    const result = await tryTo(() => I.see('Add to cart'));
```

Comment:
move tryTo() to helper as function, which accepts locator of element and returns true / false either is exists or not